doc/source/plots/lic.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import division
+
+# We will visualize a 2-D vortex with different configurations of the
+# ``line_integral_convolution`` algorithm, and we will use the 2-D vortex to
+# add motion blur to a sample image. First the necessary modules and functions
+# need to be imported.
+
+import numpy as np
+from skimage.filters import line_integral_convolution
```

Comment:
Do you think this fits better under `filters` or `draw`?  My intuition is perhaps `draw`?